test-data/unit/check-callable.test

```diff
@@ -525,7 +525,7 @@ else:
 [builtins fixtures/callable.pyi]
 
 [case testBuiltinsTypeAsCallable]
-# flags: --python-version 3.7
+# flags: --python-version 3.8
```

Comment:
I think we can just remove this; it was presumably added so that the future import is available, but the future import is now available on all supported versions.

Reply:
```suggestion
```